src/CodeGen_ARM.cpp

```diff
@@ -885,6 +883,9 @@ void CodeGen_ARM::visit(const Load *op) {
             return;
         }
 
+#if LLVM_VERSION < 37
```

Comment:
PR #1457 officially removes support for llvms < 3.7. Once that lands we can nuke the LLVM_VERSION <37 branch in this PR.